.github/workflows/release.yaml

```diff
@@ -0,0 +1,48 @@
+name: GITHUB RELEASE
+
+on:
+  push:
+    branches: [ master ]
+    paths-ignore:
+      - '**/README.md'
+  pull_request:
+    branches: [ master ]
+    paths-ignore:
+      - '**/README.md'
```

Comment:
We probably don't want to release the map on the pull request. We should only limit this to push to the master branch.

Reply:
Done!